pkg/logql/metrics.go

```diff
@@ -13,6 +14,7 @@ import (
 
 	"github.com/grafana/loki/pkg/logql/syntax"
 	"github.com/grafana/loki/pkg/logqlmodel"
+	"github.com/grafana/loki/pkg/logqlmodel/stats"
```

Comment:
extra import, line below is `logql_stats "github.com/grafana/loki/pkg/logqlmodel/stats"`